game/ui/documentmenu.h

```diff
@@ -25,6 +25,7 @@ class DocumentMenu : public Tempest::Widget {
       Flags           flg=F_None;
       };
 
+    // TODO: set default values for these
```

Comment:
The commit message of b9462aa4 (which added that line) has an explanation. In short: sometimes the script does not actually set the document font in which case we fail to load it and instead return a default font. I'm not sure whether that's the proper way of doing things and it might be beneficial to set a default font name there.